### ISSUE TITLE
Slug preview does not work with multiple tabs instances

### ISSUE THREAD

**Issue by fsavina-akqa** [kongulov/nova-tab-translatable#44]
Hello,
thanks for this useful package.

I have a model with multiple instances of NovaTabTranslatable and I get a 404 error when the slug preview is getting loaded.

The reason why this is happening is because this foreach cycle should terminate when the field is found.
Otherwise the following iterations will overwrite the value of `$field` with `null`.

```
// Kongulov\NovaTabTranslatable\Http\Controllers\FieldPreviewController, line 43

foreach ($tabs as $tab) {
  $field = collect($tab->data)->first(function($field) use ($request){
    return isset($field->attribute) &&
    $field->attribute == $request->field;
  });
} 
```

Thank you very much

**Comment by fsavina-akqa** [comment]
Hi @kongulov, 
I'm using the version [2.1.3](https://github.com/kongulov/nova-tab-translatable/releases/tag/2.1.3).

Here is a simplified version of my Nova resource fields:

```
public function fields(NovaRequest $request)
{
    return [
        NovaTabTranslatable::make([
            Text::make('Title'),

            Slug::make('Slug')->from('title'),
        ]),

        Date::make('Date')->nullable(),

        NovaTabTranslatable::make([
            Text::make('Content')->nullable(),
        ]),
    ];
}
```

Just so you now, the structure of my resource is more complex and I'm using multiple NovaTabTranslatable instances for the sake of usability and content organisation.

Thanks again for your support,
F.

**Comment by fsavina-akqa** [comment]
Hi @kongulov,
any update on this topic?

Thanks, 
F.

**Comment by fsavina** [comment]
I've create a PR in order to fix this
https://github.com/kongulov/nova-tab-translatable/pull/48

Thanks,
F.

**Comment by fsavina-akqa** [comment]
Fixed in #48
